### DISCUSSION TITLE
LF APAC开源布道者分享系列之六：建立对话 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#31] [Announcements]
# LF APAC开源布道者分享系列之六：建立对话 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/advocate-how-to-build-conversation-with-peer/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

建立对话/交流, 当然重要, 可重要的在其中是有成本的;
单单是开始对话前, 厘定问题列表就是一个不简单的事儿;

其实, 对话最容易在什么场景中展开?

相亲时...

- 双方都有不得不来的原因
- 但是, 双方目标是相同的
- 而且, 为了节省时间, 必须立即开始沟通, 以免错过真正的白马们..

所以, 在其它场景中的对话建立, 其实也是相同基准:

- 谁带有目的来的?
- 谁有明确的问题?
- 和谁拥有类似经历, 可以节省大海常识核对?
- ...

这其实也是所谓圆桌会议组建的原则, 开源布道, 并是象卖大力丸那样,
无论在哪儿, 都可以立即画个摊就开始表演...

毕竟 开源是有门槛, 有技能要求的,
对不同人群, 在不同场景中, 都值得掌握一组自己的破冰套件, 以便快速发现值得建立对话的人:

- 面对没有开发经历的普通人, 分享真实世界中开源到底在什么地方服务大家, 引发投入开源的兴趣
- 对于学生, 有兴趣进入开源的人, 分享自己进入开源的故事/路径, 给出合理忠告, 指出合理沟通渠道
- 对于厂商, 有兴趣/已经来使用开源资源的人, 分析开源成本, 商业模式, 分享成功/失败故事
- 对于...

是的, 对话的建立, 关键还是在自己, 就象俺经常说的:

```
什么是卓越?
那就是有困难, 克服困难要上;
没有困难, 制造困难也要上...
```
